contrib/vpnc-scripts/template.py

```diff
@@ -0,0 +1,16 @@
+pkgname = "vpnc-scripts"
+_commit = "811a878edf944b55eb900cb309dcf5853d475b38"
+pkgver = "0_git20231220"
+pkgrel = 0
+pkgdesc = "OpenConnect network routing script"
+maintainer = "Erica Z <zerica@callcc.eu>"
+license = "GPL-2.0-or-later"
+url = "https://www.infradead.org/openconnect/vpnc-script.html"
+source = f"https://gitlab.com/openconnect/vpnc-scripts/-/archive/{_commit}/vpnc-scripts-{_commit}.tar.bz2"
+sha256 = "128f14953966dfcfedf2c4e236ca17ed3cb4e9aa79a3205af97e25d253f087e8"
+
+
+def do_install(self):
+    self.install_file("vpnc-script", "etc/vpnc", 0o755)
+    self.install_file("vpnc-script-ptrtd", "etc/vpnc", 0o755)
+    self.install_file("vpnc-script-sshd", "etc/vpnc", 0o755)
```

Comment:
/etc seems wrong. if not /usr/bin, there is /usr/libexec , or perhaps some other /usr path (but /etc seems bad unless it's user-editable)